src/sherpa_ai/output_parsers/citation_validation.py

```diff
@@ -1,13 +1,14 @@
 import nltk
 from nltk.tokenize import sent_tokenize, word_tokenize
 
-from sherpa_ai.output_parsers.base import BaseOutputParser
+from sherpa_ai.memory import Belief
+from sherpa_ai.output_parsers.base import BaseOutputProcessor
 from sherpa_ai.output_parsers.validation_result import ValidationResult
 
 nltk.download("punkt")
```

Comment:
It's a small module in nltk that needs to be downloaded on the first run. And it has to be installed through code but not something like pip. It will be automatically cached after it's being downloaded for the first time and the caching is handled by nltk.